.github/workflows/actionlint.yml

```diff
@@ -0,0 +1,19 @@
+name: Lint GitHub Actions workflows
+
+on:
+  pull_request:
+  push:
+    branches:
+      - main
+
+jobs:
+  actionlint:
+    runs-on: ubuntu-latest
+    steps:
+      - uses: actions/checkout@v3
+      - name: Check workflow files
+        run: |
+          echo "::add-matcher::.github/actionlint-matcher.json"
+          bash <(curl https://raw.githubusercontent.com/rhysd/actionlint/main/scripts/download-actionlint.bash)
+          ./actionlint -color
```

Comment:
I originally used the pre-commit hook but because we don't use pre-commit in the frontend repository it required configuring the action there anyway.

We can constrain it to a specific version by replacing `main` with a version. That is probably wise to do :slightly_smiling_face: